pkg/network/protocols/http2/testutils.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux_bpf && test
+
+package http2
+
+import (
+	"strings"
+
+	"github.com/cilium/ebpf"
+)
+
+// CleanHTTP2Maps deletes all entries from the http2 maps. Test utility to allow reusing USM instance without carring
+// over previous data.
+func CleanHTTP2Maps() {
+	if Spec.Instance == nil {
+		return
+	}
+
+	m := Spec.Instance.(*Protocol).mgr
+	if m == nil {
+		return
+	}
+
+	for _, mapElement := range Spec.Maps {
+		if strings.HasSuffix(mapElement.Name, "http2_batches") || strings.HasSuffix(mapElement.Name, "http2_telemetry") {
+			continue
+		}
+		mapInstance, _, err := m.GetMap(mapElement.Name)
+		if err != nil {
+			continue
```

Comment:
shouldn't we at least log if we were not able to get the map?

Reply:
we can, but I'm not sure it is useful for the context of the test
that's very strange to happen, and unless the test fails, we won't see the log